extensions/ql-vscode/src/vscode-tests/cli-integration/remote-queries/run-remote-query.test.ts

```diff
@@ -259,6 +264,101 @@ describe('Remote queries', function() {
     }
   });
 
+  it('should run a variant analysis that is part of a qlpack', async () => {
+    await setVariantAnalysisLiveResultsEnabled(true);
```

Comment:
Perhaps we can move this to a nested `describe` block which has this as a `beforeEach` hook? Something like:

```ts
describe('when live results are enabled', () => {
  beforeEach(async () => {
    await setVariantAnalysisLiveResultsEnabled(true);
  });

  it('should run a variant analysis that is part of a qlpack', async () => {
    // ...
  });

  // ...
});
```

Reply:
Ooh, that's a good idea. I wasn't aware you could do nested describes with their own before/after hooks.

Reply:
This has increased the total number of lines changed quite a lot, but if you enable the mode to ignore whitespace then it'll look a lot more manageable.